.github/workflows/release-v2.yml

```diff
@@ -0,0 +1,47 @@
+name: Release v2 version
+
+on: workflow_dispatch
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./packages/core
+    env:
+      NODE_AUTH_TOKEN: ${{ secrets.npm_token }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Run Setup
+        uses: ./.github/actions/setup
+        with:
+          npm_token: ${{ secrets.npm_token }}
+      - uses: ./.github/actions/git-creds
+      - name: Check current branch is "version2"
+        run: |
+          if [[ "$(git branch --show-current)" != "version2" ]]; then
+            echo "Error: Not on 'version2' branch."
+            exit 1
+          fi
+      - name: Build
```

Comment:
Lerna doesn't support version/publish for a single package so we just need to do all the checks as usual without lerna